fix related to https://github.com/wso2/docs-apim/issues/5277 (#5278)
wso2/docs-apim: en/docs/develop/streaming-apps/http-status-codes.md

# HTTP Status Codes

When REST API requests are sent to carry out various actions, various HTTP status codes will be returned based on the state of the action (success or failure) and the HTTP method (`POST`, `GET`, `PUT`, `DELETE`) executed. The following are the definitions of the various HTTP status codes that are returned.

## HTTP status codes indicating successful delivery

<table>
<tr>
<td> Code</td>
<td>Code Summary</td>
<td> Description</td>
</tr>
<tr> 200</td>
<td>Ok</td>
<td>HTTP request was successful. The output corresponding to the HTTP request will be returned. Generally used as a response to successful <code>GET</code> and <code>PUT</code> REST API HTTP methods.</td>
</tr>
<tr> 
<td>201</td>
<td>Created</td>
<td>HTTP request was successfully processed and a new resource was created. Generally used as a response to a successful <code>POST</code> REST API HTTP method.</td>
</tr>
<tr> 
<td>204</td>
<td>No content</td>
<td>HTTP request was successfully processed. No content will be returned. Generally used as a response to a successful <code>DELETE</code> REST API HTTP method.</td>
</tr>
<tr> 
<td>202</td>
<td>Accepted</td>

> <td>HTTP request was accepted for processing, but the processing has not been completed. This generally occurs when your successful in trying to undeploy an application.</td>

Reviewer: ```suggestion
<td>HTTP request was accepted for processing, but the processing was not complete. This generally occurs when you are successful in trying to undeploy an application.</td>
```
